src/plugins/orangehrmPerformancePlugin/entity/PerformanceTracker.php

```diff
@@ -35,6 +35,8 @@ class PerformanceTracker
 {
     use DecoratorTrait;
 
+    public const STATUS_NOT_DELETED = 1;
+    public const STATUS_TRACKER_DELETED = 2;
```

Comment:
might be better if names are consistent
```suggestion
    public const STATUS_TRACKER_NOT_DELETED = 1;
    public const STATUS_TRACKER_DELETED = 2;
```
or
```suggestion
    public const STATUS_NOT_DELETED = 1;
    public const STATUS_DELETED = 2;
```